src/entity.ts

```diff
@@ -289,6 +289,32 @@ export class Entity {
 	serialize(): object {
 		return this.serializer.serialize(this);
 	}
+
+	persist() {
```

Comment:
I wonder about the method name. It kind of sounds like it would actively do some sort of persistence, although anyone thinking that should obviously pause to consider how exactly it would do that. 😛  Anyway, what do you think? `markPersisted` came to mind as a possibility. Seems like ideally it would be more specific to the `isNew` property, but I can't think of a good option.

Reply:
Method is now called `markPersisted`.